fix flake8 errors

Lot's of errors according to flake8.
https://flake8.pycqa.org/en/latest/

**I accept PR's with single file's optimized.**

Thanks for you help!

You can test it after installing the requirements via:

`make flake8`

or single files like this
`cd ../auto-backup`
`cd ..`
`flake8 auto-backup/backups/ftp.py`


## Open errors:

```
./cronjobs.py:1:1: F401 'encodings.utf_8' imported but unused
./cronjobs.py:7:1: E302 expected 2 blank lines, found 1
./cronjobs.py:50:40: W292 no newline at end of file
./main.py:5:1: F401 'time.sleep' imported but unused
./main.py:12:1: E303 too many blank lines (3)
./main.py:55:1: E302 expected 2 blank lines, found 1
./main.py:59:1: E302 expected 2 blank lines, found 1
./main.py:133:1: W293 blank line contains whitespace
./main.py:134:93: W291 trailing whitespace
./main.py:138:44: W292 no newline at end of file
./backups/ftp.py:3:1: E402 module level import not at top of file
./backups/ftp.py:4:1: E402 module level import not at top of file
./backups/ftp.py:5:1: E402 module level import not at top of file
./backups/ftp.py:6:1: E402 module level import not at top of file
./backups/ftp.py:15:101: E501 line too long (135 > 100 characters)
./backups/ftp.py:62:1: E305 expected 2 blank lines after class or function definition, found 1
./backups/googledrive.py:18:101: E501 line too long (135 > 100 characters)
./backups/googledrive.py:41:101: E501 line too long (116 > 100 characters)
./server/main.py:1:1: F401 'flask.request' imported but unused
./server/main.py:9:1: E302 expected 2 blank lines, found 1
./server/main.py:10:1: W293 blank line contains whitespace
./server/main.py:14:101: E501 line too long (125 > 100 characters)
./server/main.py:24:101: E501 line too long (105 > 100 characters)
./server/main.py:29:25: F841 local variable 'file_id' is assigned to but never used
```

--- a/backups/ftp.py
+++ b/backups/ftp.py
@@ -1,5 +1,3 @@
-import sys
-sys.path.append('/Users/patrickgerard/Documents/GitHub/auto-backup')
 import settings
 import logging
 import datetime
@@ -12,7 +10,9 @@ class Ftp():
         self.filename = filename
         self.uploadpath = uploadpath
         self.time = datetime.datetime.now()
-        logging.basicConfig(filename='logs.log', filemode='a+', format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO,
+        logging.basicConfig(filename='logs.log', filemode='a+', format='%(asctime)s \
+                            - %(levelname)s \
+                            - %(message)s', level=logging.INFO,
                             datefmt='%d-%m-%Y %H:%M:%S')
         try:
             self.connection = FTP(host=settings.FTP_URL + settings.FTP_PORT)
@@ -58,6 +58,7 @@ class Ftp():
         self.connection.dir(dir_list.append)
         directories = [dir.split(" ")[-1] for dir in dir_list if dir.split(" ")[0] == "drwxr-xr-x"]
         return directories
+
 
 if __name__ == "__main__":
     ftpcon = Ftp()

--- a/backups/googledrive.py
+++ b/backups/googledrive.py
@@ -15,7 +15,9 @@ class Googledrive():
         self.parent_folder = settings.GOOGLEDRIVE_FOLDER
         self.creds = ServiceAccountCredentials.from_json_keyfile_name(settings.GOOGLEDRIVE_API)
         self.time = datetime.datetime.now()
-        logging.basicConfig(filename='logs.log', filemode='a+', format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO,
+        logging.basicConfig(filename='logs.log', filemode='a+', format='%(asctime)s \
+                            - %(levelname)s \
+                            - %(message)s', level=logging.INFO,
                             datefmt='%d-%m-%Y %H:%M:%S')
         logging.info('Googledrive backer started')
 
@@ -38,7 +40,8 @@ class Googledrive():
             media = MediaFileUpload(self.filename)
             file = service.files().create(body=file_metadata, media_body=media,
                                           fields='id').execute()
-            logging.warning(F'File {self.filename.split("/")[1]} uploaded to Googledrive with ID: {file.get("id")}')
+            logging.warning(F'File {self.filename.split("/")[1]} uploaded to Googledrive with ID: \
+                            {file.get("id")}')
 
         except HttpError as error:
             logging.error(F'An error occurred: {error}')

--- a/cronjobs.py
+++ b/cronjobs.py
@@ -1,8 +1,8 @@
-from encodings import utf_8
 from crontab import CronTab
 import logging
 import getpass
 import os
+
 
 class Croncontroller():
 

--- a/main.py
+++ b/main.py
@@ -2,11 +2,9 @@ import argparse
 import subprocess
 import os
 from threading import Thread
-from time import sleep
 
 import cronjobs
 from backups import googledrive, ftp
-
 
 
 def upload_single(working_dir: str, path: str, storage: str):
@@ -52,9 +50,11 @@ def compress(working_dir: str, path: str):
     except Exception as e:
         print(e)
 
+
 def _server(c):
     from server import main
     main.start()
+
 
 def ngrok():
     subprocess.call('ngrok http 5000', shell=True)
@@ -130,8 +130,7 @@ if __name__ == "__main__":
             print("To see all logs open logs.log")
         except Exception as e:
             print(f"No logs saved: {e}")
-    
-    # current problem: when uploading another file, another Thread of expose will be started 
+    # current problem: when uploading another file, another Thread of expose will be started
     # free version is limited to 1
     # it keeps working but Exceptions show up
     serv = Thread(target=_server, args=[5]).start()

--- a/server/main.py
+++ b/server/main.py
@@ -1,4 +1,4 @@
-from flask import Flask, render_template, request, send_from_directory
+from flask import Flask, render_template, send_from_directory
 import logging
 import sys
 import click
@@ -6,12 +6,14 @@ import click
 log = logging.getLogger('werkzeug')
 log.disabled = True
 
+
 def start():
-    
+
     cli = sys.modules['flask.cli']
 
     # put your own message here
-    cli.show_server_banner = lambda *x: click.echo("Your logs and backup statistics will be visualized here: http://0.0.0.0")
+    cli.show_server_banner = lambda *x: click.echo("Your logs and backup statistics will \
+                                                   be visualized here: http://0.0.0.0")
 
     app = Flask(__name__,
                 static_folder='/storage')
@@ -21,12 +23,12 @@ def start():
         data = {}
         with open('./logs.log', 'r+') as logs:
             for n, line in enumerate(logs.readlines()):
-                timestamp, status, msg = line.split(' - ')[0], line.split(' - ')[1], line.split(' - ')[2]
+                timestamp, status, msg = line.split(' - ')[0], line.split(' - ')[1],
+                line.split(' - ')[2]
                 if status == 'WARNING':
                     if "Googledrive" in msg:
                         # improvement: create ahref to googledrive to view with
                         # https://drive.google.com/file/d/<id>/view
-                        file_id = msg.split('ID: ')[1]
                         data[n] = [timestamp, status, msg]
                     else:
                         data[n] = [timestamp, status, msg]

